dashboard/src/components/EventsListItem.js

```diff
@@ -55,19 +63,48 @@ class EventListItem extends React.Component {
     // eslint-disable-next-line react/forbid-prop-types
     classes: PropTypes.object.isRequired,
     Chevron: PropTypes.func.isRequired,
+    Disclosure: PropTypes.func.isRequired,
   };
 
   static defaultProps = {
     Chevron: chevronIcon,
+    Disclosure: disclosureIcon,
+  };
+
+  state = { anchorEl: null };
+
+  onClose = () => {
+    this.setState({ anchorEl: null });
+  };
+
+  handleClick = event => {
+    this.setState({ anchorEl: event.currentTarget });
+  };
+
+  silenceEntity = entity => () => {
+    console.log(entity);
```

Comment:
🤔 linter should not be passing with these present. Regardless- by convention- add `eslint-disable-next-line` and ideally switch to info for the time being.